in_toto/runlib.py

```diff
@@ -784,6 +798,10 @@ def in_toto_record_stop(step_name, product_list, signing_key=None,
     base_path (optional): A path relative to which artifacts are recorded.
         Default is the current working directory.
 
+    byproducts (optional): A dictionary that lists byproducts of the link
```

Comment:
Curious, at some point here, we stop needing in-toto-record, correct? All this information is available and could be plugged into a `Link` object manually as well?

Reply:
Makes sense, thanks!